index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+    
+</head>
+<body>
+    <header class="header">
+        <div class="header-contain">
+            <div class="logo">My Logo</div>
+            <div>
+                <img class="vector" src="./assets/img/Union.png" alt="">
+                
+            </div>
+        </div>
+    </header>
+    <section>
+        <div class="main__section">
+            <div>
+            <h1 class="title">I'm M.Zaki <br> glad to see <br> you!</h1>
+            <p class="text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <p class="connect-btn">LET'S CONNECT</p>
+            <div class="media-link">
```

Comment:
- [ ] If you notice `Let's connect` is a link that should direct you to contact areas like social icons or contact form :smirk: . This implies that the text should be a link and hence `a` tag should be used instead of `<h2>` tag. For [Semantic purposes](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwi56cjW38D8AhX0UaQEHVp7DLEQFnoECAsQAQ&url=https%3A%2F%2Fwww.w3schools.com%2Fhtml%2Fhtml5_semantic_elements.asp&usg=AOvVaw38SC2etR0JeORCKyF1fqbU), kindly ensure that the text is a link to any of any of suggestions( social media or any contact element).